### DISCUSSION TITLE
Display Name Removal

### DISCUSSION THREAD

**Discussion by IstalriSkolir** [community/community#133402] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How do I hide my display name? My username (Istalri Skolir Productions) is fine but my display name is using my real name which I'd rather not have shown publicly and would prefer not to show on my profile or integrations. How can it be removed so that is its shown anywhere on GitHub or platforms that I integrate my account onto?

**Comment by davevad93** [comment, marked as answer]
Hi @JohnWolfson ,

`Istalri Skolir Productions` is actually your "full name" on GitHub while `JohnWolfson` is your username. If that's your real name and you don't want it to be displayed publicly you'll have to change it by going to `Settings` => `Account` => `Change Username`:

![1](https://github.com/user-attachments/assets/49e62607-50b6-4cfe-8fb4-00edb7444d81)

Before doing so, take a look at the [unintended side effects](https://docs.github.com/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/changing-your-github-username) of changing your username on GitHub.

**Comment by IstalriSkolir** [comment]
Ah, I hadn't realised I had got the two mixed up, its been a long time since I made my account. Thank you for clarifying!
